### PR TITLE
Avtar badge

### PR DESCRIPTION
**Pull request by manojncsoni007**
completed avatar and badge components

### CODE REVIEW

File: Components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+* {
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
+}
```
Comment (arpit01923) [inline Components/avatar/avatar.css:5]: put it in common css file

File: Components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    width: 90%;
+    padding: 1rem;
+    margin: auto;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+.avatar-container {
+    display: flex;
```
Comment (arpit01923) [inline Components/avatar/avatar.css:17]: if flex property is used again and again put it in common css file

File: Components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url(../avatar/avatar.css);
+* {
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
+}
```
Comment (arpit01923) [inline Components/badge/badge.css:6]: add this to common css file

File: Components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+.badge-container {
+    padding: 2rem;
+    display: flex;
```
Comment (arpit01923) [inline Components/badge/badge.css:10]: put flex property in common css file

File: Components/badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+.avatar-badge {
+    position: absolute;
+    padding: 0.5rem;
+    border-radius: 50%;
+    bottom: 9%;
+    right: 0;
+}
+
+.green {
```
Comment (arpit01923) [inline Components/badge/badge.css:80]: put color in root element and put that root element in common css file

**Review by arpit01923 — APPROVED** [review]
minor conflicts

**Comment by manojncsoni007** [comment]
Screenshot (163)
Screenshot (164)

File: pages/docs.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        <div class="navbar-menu">
+            <a href="">Installation</a>
+            <a href="docs.html">Docs</a>
+        </div>
+    </nav>
+    <div class="docs-container">
+        <div class="docs-sidebar-container">
+            <aside class="docs-sidebar">
+                <h3 class="docs-sidebar-title">Components</h3>
+                <div class="docs-sidebar-link-container">
```
Comment (Aditi0505) [inline pages/docs.html:27]: Suggestion: You can go for a shorter class name. (Eg. sidebar-links or links-container ). That would be more readable.

**Review by Aditi0505 — APPROVED** [review]
Added one suggestion. Aprit has suggested some good changes. Rest looks good!

**Pull request merged**